downstream/modules/automation-hub/proc-create-synclist.adoc

```diff
@@ -2,34 +2,32 @@
 // obtaining-token/master.adoc
 [id="proc-create-synclist"]
 
-= Creating a synclist of Red Hat Certified collections
+= Creating a synclist of Red Hat {CertifiedName}
 
-You can create a synclist of curated Red Hat Certified collections in {HubName} on console.redhat.com. 
-Your synclist repository is located under menu:Automation Hub[Repo Management], which is updated whenever you choose to manage content within Red Hat Certified collections.
+You can create a synclist of curated {CertifiedCon} in {HubNameMain} on {Console}. 
+Your synclist repository is located under menu:Automation Hub[Repo Management], which is updated whenever you choose to manage content within {CertifiedName}.
 
-By default, all Red Hat Certified collections are included in your initial organization synclist.
+All {CertifiedName} are included by default in your initial organization synclist.
 
 .Prerequisites
 
 * You have a valid {PlatformNameShort} subscription.
-* You have Organization Administrator permissions for console.redhat.com.
+* You have Organization Administrator permissions for {Console}.
 * Ensure that the following domain names are part of either the firewall or the proxy's allowlist for successful connection and download of collections from {HubName} or Galaxy server:
 ** `galaxy.ansible.com`
 ** `cloud.redhat.com`
-** `console.redhat.com`
+** `{Console}`
 ** `sso.redhat.com`
-* {HubNameStart} resources are stored in Amazon Simple Storage.
+* {HubNameMain} resources are stored in Amazon Simple Storage.
 Add the following domain name to the allow list:
 ** `automation-hub-prd.s3.us-east-2.amazonaws.com`
 * SSL inspection must be disabled either when using self signed certificates or for the Red Hat domains.
 
-
 .Procedure
 
-. Log in to console.redhat.com.
+. Log in to `{Console}`.
 . Navigate to menu:Automation Hub[Collections].
 . Use the toggle switch on each collection to determine whether to exclude it from your synclist.
 
-
-When you finish managing collections for your synclist, you can navigate to menu:Automation Hub[Repo Management] to initiate the remote repository sync to your local {HubName}. 
-If your remote repository is already configured, you can manually sync Red Hat Certified collections to your local {HubName} to update the collections content that you made available to local users.
+When you finish managing collections for your synclist, navigate to menu:Automation Hub[Repo Management] to initiate the remote repository synchronization to your {PrivateHubName}. 
```

Comment:
Should this be step 4? Is this synchronization  required or optional? Is this the only place where we see this instruction or is there a procedure for this?